### PR TITLE
support for vs2022

### PR DESCRIPTION
**Pull request by ihcaesar**
amd64 only

### CODE REVIEW

File: src/AutoMerge.Tests/AutoMerge.Tests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>net48</TargetFramework>
```
Comment (aalexr) [inline src/AutoMerge.Tests/AutoMerge.Tests.csproj:4]: Could you please add with condition on $(VisualStudioVersion) it like in the line below so we don't break it for VS2017.

File: src/AutoMerge/AutoMerge.csproj [hunk 2]
```diff
@@ -1,264 +1,264 @@
...
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{726ED85E-2274-4D95-B822-B2CFE2CE44B9}</ProjectGuid>
+    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>AutoMerge</RootNamespace>
+    <AssemblyName>AutoMerge</AssemblyName>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>Key.snk</AssemblyOriginatorKeyFile>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```
Comment (aalexr) [inline src/AutoMerge/AutoMerge.csproj:24]: Same. Please, add with condition on $(VisualStudioVersion) it like in the line below so we don't break it for VS2017.

File: src/AutoMerge/source.extension.vsixmanifest [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
 	<Metadata>
-		<Identity Id="AutoMerge.VS2017.F05BAC3E-6794-4A9E-9EE7-1B8A200778EE" Version="0.2.6.9" Language="en-US" Publisher="Kulikov Denis" />
-		<DisplayName>Auto Merge for Visual Studio 2017</DisplayName>
+		<Identity Id="AutoMerge.VS2019.F05BAC3E-6794-4A9E-9EE7-1B8A200778EE" Version="0.2.6.10" Language="en-US" Publisher="Kulikov Denis" />
```
Comment (aalexr) [inline src/AutoMerge/source.extension.vsixmanifest:4]: I believe this file should not be changed, introducing VS-specific manifest "source.vs17.0.extension.vsixmanifest" is enough.

**Review by aalexr — CHANGES_REQUESTED** [review]
I believe you forgot adding VS2022 specific libs (dll).

File: src/AutoMerge/source.extension.vsixmanifest (hunk 3 above)
Comment (ihcaesar) [reply src/AutoMerge/source.extension.vsixmanifest:4]: This file is automatically changed by the build process - so no need to undo.

**Comment by aalexr** [comment]
Thank you for contribution! I think it is ready to merge. Now I will try to figure out how to publish that to VS Extensions Store

**Pull request merged**